README.md

```diff
@@ -62,15 +62,123 @@ For running integration tests in Anvil node, run `make anvil-tests`. This runs t
+```shell
+forge script script/bytecode-deploy/105_SetUpgradableMSCAFactoryPlugins.s.sol  -vvvv --slow --broadcast --force --multi
+forge script script/bytecode-deploy/106_StakeUpgradableMSCAFactory.s.sol  -vvvv --slow --broadcast --force --multi
+```
+#### Verify contracts on block explorers.
+   
+Use the json files in `script/bytecode-deploy/standard-json-input` to verify smart contracts. If the smart contract has
+constructor arguments, it's recorded in `*_constructor_arg` files. Select the following options in block explorers verify contract UI:
+* Compiler Type: `Solidity(Standard-Json-Input)`
+* Compiler Version: `v0.8.24`
+* License Type: `GNU GPLv3`
+
+### Deploy & Verify EPv0.6 SponsorPaymaster
+Deploy, setup and verify SponsorPaymaster compatible with ERC-4337 v0.6.
+
+#### Run deploy command
+```shell
+forge script script/bytecode-deploy/107_DeploySponsorPaymasterEPv06.s.sol  -vvvv --slow --broadcast --force --multi
+```
+
+#### Setup SponsorPaymaster
+
+Need to use block explorer to run `addStake` and `setVerifyingSigner` from owner address.
+
+#### Verify SponsorPaymaster
+   
+Use `script/bytecode-deploy/standard-json-input/SponsorPaymasterEPv06.json` to verify. Select the following options 
+in block explorers verify contract UI:
+* Compiler Type: `Solidity(Standard-Json-Input)`
+* Compiler Version: `v0.8.17`
+* License Type: `GNU GPLv3`
+
+### Deploy & Verify EPv0.7 SponsorPaymaster
+Deploy, setup and verify SponsorPaymaster compatible with ERC-4337 v0.7.
+
+#### Run deploy command
+    1. Replace the placeholders and run the command in `script/cmd/DeploySponsorPaymaster`.
+    2. Replace the placeholders and run the command in `script/cmd/DeploySponsorPaymasterProxy`.
+
+#### Setup SponsorPaymaster
+
+Need to use block explorer to run `addStake` and `setVerifyingSigner` from owner address.
+
+#### Verify SponsorPaymaster
+   Use `script/verify/SponsorPaymaster.json` and `script/verify/SponsorPaymasterProxy.json` to verify. Select the 
+following options in block explorers verify contract UI:
+* Compiler Type: `Solidity(Standard-Json-Input)`
+* Compiler Version: `v0.8.24`
+* License Type: `GNU GPLv3`
+
+### Deploy & Verify SingleOwnerMSCAFactory
+Deploy, setup and verify SingleOwnerMSCAFactory compatible with ERC-4337 v0.6 and ERC-6900 v0.7.
+
+#### Run deploy commands:
+```shell
+# Deploy PluginManagerEPv06
+forge script script/bytecode-deploy/108_DeployPluginManagerEPv06.s.sol  -vvvv --slow --broadcast --force --multi
+
+# Deploy SingleOwnerMSCAFactoryEPv06
+forge script script/bytecode-deploy/109_DeploySingleOwnerMSCAFactoryEPv06.s.sol  -vvvv --slow --broadcast --force --multi
+
+# Deploy ColdStorageAddressBookPluginEPv06
+forge script script/bytecode-deploy/110_DeployColdStorageAddressBookPluginEPv06.s.sol  -vvvv --slow --broadcast --force --multi
+```
+
+#### Verify contracts on block explorers.
```

Comment:
There seems to be a lot of duplication in these verification sections. What do you think about having just one section for all of them?